lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -357,6 +357,20 @@ ResolveTypeAlias(SwiftASTContext *module_holder,
                  swift::Demangle::NodePointer node,
                  bool prefer_clang_types = false) {
   LLDB_SCOPED_TIMER();
+  auto resolve_clang_type = [&]() {
+    if (node->getNumChildren() == 2 && node->getChild(0)->hasText() &&
+        node->getChild(0)->getText() == swift::MANGLING_MODULE_OBJC &&
+        node->getChild(1)->hasText()) {
+      // Resolve the typedef within the Clang debug info.
+      auto clang_type =
+          LookupClangForwardType(module_holder, node->getChild(1)->getText());
+      if (!clang_type)
+        return CompilerType();
```

Comment:
convention question: do we prefer `return {}` here?

Reply:
As the person who started this convention: yes :-)

Reply:
I prefer that as well, but can't do it because C++ can't infer the return type. I guess I can specify the return type so I can return `{}`